third_party/move/move-compiler-v2/tools/testdiff/src/main.rs

```diff
@@ -106,6 +106,10 @@ static UNIT_PATH_REMAP: Lazy<BTreeMap<String, String>> = Lazy::new(|| {
         "visibility-checker/v1-typing".to_string(),
         "typing".to_string(),
     );
+    map.insert(
+        "recursive-instantiation-checker/v1-tests".to_string(),
```

Comment:
Better if we can avoid such spurious test renaming.

Reply:
Which part do you think is spurious? "recursive-instantiation" and "instantiation_loops" mean the same thing here.

Reply:
If there was a V1 test directory called "foo", we should just call it "foo" in V2.  No need to rename them, though Wolfgang loves to do so.

Reply:
I changed the directory to `cyclic-instantiation-checker' so it aligns with the name of the checker. "Instantiation loops" doesn't appear anywhere in V2, so I'll just keep in the renaming map.